db/blip_handler.go

```diff
@@ -1357,6 +1381,13 @@ func (bh *blipHandler) sendProveAttachment(sender *blip.Sender, docID, name, dig
 	if !bh.sendBLIPMessage(sender, outrq) {
 		return ErrClosedBLIPSender
 	}
+	messageCPUtime := time.Since(startTime).Milliseconds()
+	messBytes, err := outrq.Body()
+	if err == nil && bh.blipContextDb != nil {
+		bytes := len(messBytes)
+		stat := CalculateComputeStat(int64(bytes), messageCPUtime)
+		bh.blipContextDb.DbStats.Database().SyncProcessCompute.Add(stat)
+	}
```

Comment:
as above, I think this comes from `processRev`, can you verify my assumption and add a comment?